### ISSUE TITLE
Silo uptime and local time

### ISSUE THREAD

**Issue by DixonDs** [OrleansContrib/OrleansDashboard#42]
The silo uptime time seems to wrong. See the screenshot:
![uptime_time](https://cloud.githubusercontent.com/assets/1036478/24943948/7eaec3b4-1f61-11e7-9927-245703ea3f1c.png)

The screenshot was taken during the deployment when the first silo just restarted. I believe it has to do with the local timezone of the PC where the screenshot was taken - it is UTC+3

It is still 1.3.1, maybe it is not an issue in newer versions.

**Comment by richorama** [comment]
Thanks for reporting, I'll look into it today.

**Comment by richorama** [comment]
I'm having some trouble reproducing this. Are the silos and browser in different timezones?

**Comment by DixonDs** [comment]
Yes, they are.

**Comment by DixonDs** [comment]
I think in this line https://github.com/OrleansContrib/OrleansDashboard/blob/26c386b295c45f793746f1b1d1141693f3ccc42e/App/silos/host-table.jsx#L23 it should be something like follows:
```
<td>up for {humanize(new Date().getTime() + new Date().getTimezoneOffset()*60*1000 - new Date(silo.startTime).getTime(), { round: true, largest: 2  })}</td>
```

**Comment by richorama** [comment]
My understanding is that `getTime()` gives you a UTC timestamp, but there's obviously something going wrong. Maybe the silo time is in UTC, but JavaScript thinks it's local.

**Comment by dsarfati** [comment]
We are running ours in a kubernetes cluster in azure and all of our silos indicate that they have been up for 48 years using 2.0 beta :)

**Comment by richorama** [comment]
Impressive!

**Comment by SebastianStehle** [comment]
Solved in 2.0 branch.
